lib/resources/styles.css

```diff
@@ -462,6 +462,7 @@ h1 .category {
   vertical-align: middle;
 }
 
+/* The badge under a declaration for things like "const", "read-only", etc. and for the badges inline like Null safety*/
```

Comment:
I think it would be beneficial to link to the items in the backend code, I think in feature.dart?

Reply:
Looks good.